Auth0.Net/Auth0.Net.csproj

```diff
@@ -40,9 +40,8 @@
       <SpecificVersion>False</SpecificVersion>
       <HintPath>..\packages\Newtonsoft.Json.6.0.6\lib\net35\Newtonsoft.Json.dll</HintPath>
     </Reference>
-    <Reference Include="RestSharp, Version=105.0.0.0, Culture=neutral, processorArchitecture=MSIL">
-      <SpecificVersion>False</SpecificVersion>
-      <HintPath>..\packages\RestSharp.105.0.1\lib\net35\RestSharp.dll</HintPath>
+    <Reference Include="RestSharp">

```

Comment:
@siacomuzzi I just noticed that my change removed the more specific reference to the `RestSharp` assembly (i.e. no more `Version`, `Culture`, `processorArchitecture` attributes).  If this is a concern I can add them back in.